text_extensions_for_pandas/array/tensor.py

```diff
@@ -103,7 +105,7 @@ class TensorArray(pd.api.extensions.ExtensionArray, TensorOpsMixin):
     Each tensor must have the same shape.
     """
 
-    def __init__(self, values: Union[np.ndarray, Sequence[np.ndarray]],
+    def __init__(self, values: Union[np.ndarray, Sequence[np.ndarray], Any],
```

Comment:
I couldn't figure out a way to make a type hint that would satisfy the condition `np.isscalar` below, so used `Any` here for now. Any ideas @frreiss ?  This was needed for a Pandas 1.0.x test, and it does make sense that one might construct an array from a single value.

Reply:
I think `Any` is the best solution here. If you really want to, you can build a "object for which `np.isscalar()` returns true" type using the facilities in [the `typing` package](https://docs.python.org/3/library/typing.html). But that option strikes me as overkill.